### PR TITLE
Remove scenarios from daml-studio docs

### PR DESCRIPTION
**Pull request by akrmn**
### Pull Request Checklist

- [ ] Read and understand the [contribution guidelines](https://github.com/digital-asset/daml/blob/main/CONTRIBUTING.md)
- [ ] Include appropriate tests
- [ ] Set a descriptive title and thorough description
- [ ] Add a reference to the [issue this PR will solve](https://github.com/digital-asset/daml/issues), if appropriate
- [ ] Include changelog additions in one or more commit message bodies between the `CHANGELOG_BEGIN` and `CHANGELOG_END` tags
- [ ] Normal production system change, include purpose of change in description
- [ ] If you mean to change the status of a component, please make sure you keep [the Component Status page](https://github.com/digital-asset/daml/blob/main/docs/source/support/component-statuses.rst) up to date.

NOTE: CI is not automatically run on non-members pull-requests for security
reasons. The reviewer will have to comment with `/AzurePipelines run` to
trigger the build.

### CODE REVIEW

File: docs/source/daml/daml-studio/daml/Abort.daml [hunk 1]
```diff
@@ -1,11 +1,16 @@
...
 -- SPDX-License-Identifier: Apache-2.0
 
-
 module Abort where
 
-{-
-abortTest = scenario do
+import Daml.Script hiding (script)
+
+-- | This lets us type check 'abortTest' without letting it fail on `daml test`
```
Comment (akrmn) [inline docs/source/daml/daml-studio/daml/Abort.daml:8]: a bit hacky, but I think it's better than our current approach of keeping failing scenarios in block comments. 

If users actually want to write failing scripts to test that bad cases are indeed forbidden, we might want to add something like `scriptShouldFail`, perhaps taking an argument for (a substring of/a predicate on) the expected error message?

Comment (cocreature) [reply docs/source/daml/daml-studio/daml/Abort.daml:8]: yeah it has come up a few times but so far we haven’t implemented something. Part of what makes this difficult is that Daml Script also works over the gRPC API and that does not expose exceptions in a structured fashion so this gets super hacky. But I think we might be better off adding a Daml Studio only function for this than doing nothing. Anyway for your PR that seems like a fine solution :+1:

**Review by cocreature — APPROVED** [review]
Thank you!

File: docs/source/daml/daml-studio/daml/Abort.daml (hunk 1 above)
Comment (akrmn) [reply docs/source/daml/daml-studio/daml/Abort.daml:8]: ah yeah I see how that's tricky. Good to know it's in our radar though, thanks!

**Pull request merged**